README.md

```diff
@@ -25,9 +26,8 @@ To get a local copy up and running follow these simple example steps.
 - Clone or download this repo on your machine
   
   ```git clone git@github.com:Kalunge/Blog-app.git```
-- cd into project directory![145261309-cf2fa744-ddfb-4c37-bf6b-5a5ebf6ea28d](https://user-images.githubusercontent.com/50773868/145261402-cc401dbf-f42f-4b09-b291-5605678952ee.png)
-
-  
+- cd into project directory
+ 
```

Comment:
- [ ] The command `rake db:reset` gives me the error. The seed data has some issue, giving the not null violation error. Please fix the seed data.

![Screenshot from 2021-12-09 19-56-48](https://user-images.githubusercontent.com/22434660/145415036-2f41d425-2af9-48b0-9089-501e00f5af99.png)

Reply:
- [ ] This issue has not been well addressed. I still am getting issues with the seed data. Kindly fix this and you are good to go. :pray: 

![Screenshot from 2021-12-10 11-13-21](https://user-images.githubusercontent.com/64894927/145542106-e02a9418-211c-4d45-9c71-cc4ffc3c6488.png)

![Screenshot from 2021-12-10 11-21-08](https://user-images.githubusercontent.com/64894927/145542156-5e9becff-e1f6-4213-a95f-b4cdbba15a87.png)